hr_timesheet_sheet_no_negative/models/hr_timesheet_sheet.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models, _
+
+
+class SheetLine(models.TransientModel):
+    _inherit = 'hr_timesheet.sheet.line'
+
+    def _negative_unit_amount(self):
+        if not self.sheet_id.company_id:
+            if self._context.get('company_id'):
```

Comment:
I read that `self._context` is deprecated, prefer to use `self.env.context` instead.